examples/gadgets/operators/otel_metrics/README.md

```diff
@@ -0,0 +1,19 @@
+# Open Telemetry Metrics Exporter Operator
+
+This example shows how to export metrics using OpenTelemetry / Prometheus.
+
+### How to run
+
+```bash
+$ go run -exec sudo .
+```
+
+### What it does
+
+<!-- markdown-link-check-disable-next-line -->
+By default, the operator exposes the metrics on http://ip:2224/metrics. This demo
```

Comment:
```suggestion
By default, the operator exposes the metrics on http://localhost:2224/metrics. This demo
```

?

Reply:
Standard is "0.0.0.0:2224" - should we change it to be 127.0.0.1 as well? Otherwise I would assume it was _only_ listening to localhost when reading the docs.

Reply:
I think 0.0.0.0 is better, then what about using http://<ip>:2224 or something like that to highlight it has to be changed in order to work?